easier68k/core/opcodes/move.py

```diff
@@ -64,6 +68,48 @@ def execute(self, simulator: M68K):
         # and set the value
         self.dest.set_value(simulator, src_val, val_length)
 
+        # increment the program counter by the length of the instruction (1 word)
+        to_increment = OpSize.WORD.value
+        print(to_increment)
+
+        if self.src.mode in [EAMode.Immediate]:
+            # add the length of the size of the operation, in words
+            if self.size is OpSize.BYTE:
+                print('adding word val')
+                to_increment += OpSize.WORD.value
+            else:
+                print('adding ')
+                print(self.size.value)
+                to_increment += self.size.value
+
+        # if followed by a long addr, add the length of the long
+        if self.src.mode in [EAMode.AbsoluteLongAddress]:
+            to_increment += OpSize.LONG.value
+
+        # same goes with a word
+        if self.src.mode in [EAMode.AbsoluteWordAddress]:
+            to_increment += OpSize.WORD.value
+
+        # repeat for the dest
+        if self.dest.mode in [EAMode.AbsoluteLongAddress]:
+            print('adding long value')
+            to_increment += OpSize.LONG.value
+
+        if self.dest.mode in [EAMode.AbsoluteWordAddress]:
+            to_increment += OpSize.WORD.value
+
+        # get the current program counter
+        pc_val = simulator.get_program_counter_value()
+        print(pc_val)
+        print(to_increment)
+
+        # assert False
+
+        # set the program counter value
+        simulator.increment_program_counter(to_increment)
+
+        print(simulator.get_program_counter_value())
```

Comment:
Do we want to remove this print? (I know it doesn't show up but probably a good idea to remove it). Same for the ones up above.